engine/flink/components-api/src/main/scala/pl/touk/nussknacker/engine/flink/api/typeinformation/FlinkBaseTypeInfoRegister.scala

```diff
@@ -0,0 +1,62 @@
+package pl.touk.nussknacker.engine.flink.api.typeinformation
+
+import org.apache.flink.api.common.typeinfo.{TypeInfoFactory, TypeInformation, Types}
+import org.apache.flink.api.java.typeutils.TypeExtractor
+
+import java.lang.reflect.Type
+import java.time.{LocalDate, LocalDateTime, LocalTime}
+import java.util
+
+object FlinkBaseTypeInfoRegister {
+
+  private[typeinformation] case class Base[T, K <: TypeInfoFactory[T]](klass: Class[T], factoryClass: Class[K])
+
+  private[typeinformation] val baseTypes = List(
+    Base(classOf[LocalDate], classOf[LocalDateTypeInfoFactory]),
+    Base(classOf[LocalTime], classOf[LocalTimeTypeInfoFactory]),
+    Base(classOf[LocalDateTime], classOf[LocalDateTimeTypeInfoFactory]),
+  )
+
+  def makeSureBaseTypesAreRegistered(): Unit =
```

Comment:
It is more idiomatic to call this `ensure..`. Compare usages of both words in the current symbols.